python/pyspark/sql/connect/client/logging.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os
+from typing import Optional
+
+__all__ = [
+    "logger",
```

Comment:
```suggestion
```

I suspect `logger` isn't an API?

Reply:
logger isn't an API, yeah. I didn't import it in __init__, but probably best to drop it here too.